app/src/main/java/com/example/calculadoramvp/mvp/model/CalculatorModel.java

```diff
@@ -1,8 +1,57 @@
 package com.example.calculadoramvp.mvp.model;
 
+import static com.example.calculadoramvp.Utils.EMPTY_STRING;
+
 public class CalculatorModel {
 
-    private int operando_1;
-    private String operador;
-    private float resultado;
+    private String operand_1;
```

Comment:
change the name plase. use the standars without "_"
In this case you can use "firstOperand"

Reply:
done